pkg/controller/worker/machines.go

```diff
@@ -114,23 +115,57 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 			Image:   machineImage,
 		})
 
-		volumeSize, err := worker.DiskSize(pool.Volume.Size)
+		disks := make([]map[string]interface{}, 0)
+
+		generateDiskFuc := func(volume v1alpha1.Volume) (map[string]interface{}, error) {
```

Comment:
Any reason for this to be an anonymous function? Maybe move it outside as regular function, wdyt?

Reply:
Accept